mpl/comm_group.hpp

```diff
@@ -274,7 +274,7 @@ namespace mpl {
       return MPI_SUCCESS;
     }
 
-    static int isend_irecv_cancel(void *state, int complete) { return MPI_SUCCESS; }
+    static int isend_irecv_cancel([[maybe_unused]] void *state, [[maybe_unused]] int complete) { return MPI_SUCCESS; }
```

Comment:
Ok. `isend_irecv_cancel` is intentionally a no-op. Parameters are technically required but intentionally not used.